### PR TITLE
copr: vectorize Sign Sqrt

### PR DESCRIPTION
**Pull request by Fullstop000**
PCP https://github.com/tikv/tikv/issues/5751

Signed-off-by: Fullstop000 <fullstop1005@gmail.com>

###  What have you changed?

Vectorize `sign` `sqrt`

###  What is the type of the changes?

- Improvement
###  How is the PR tested?

- Unit test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No

###  Does this PR affect `tidb-ansible`?

No

### CODE REVIEW

**Comment by iosmanthus** [comment]
Thanks a lot!

**Comment by breeswish** [comment]
@Fullstop000 For the moment copr-test has some issues and is disabled. We will re-open it soon. You can try it tonight.

File: components/tidb_query/src/rpn_expr/impl_math.rs [hunk 1]
```diff
@@ -180,6 +180,32 @@ fn abs_decimal(arg: &Option<Decimal>) -> Result<Option<Decimal>> {
...
+}
+
+#[inline]
+#[rpn_fn]
+fn sqrt(arg: &Option<Real>) -> Result<Option<Real>> {
+    Ok(arg.and_then(|n| {
+        if *n < 0f64 {
+            None
+        } else {
+            Some(Real::from(n.sqrt()))
```
Comment (breeswish) [inline components/tidb_query/src/rpn_expr/impl_math.rs]: Prefer to use `Real::new()` to avoid the NaN case

**Comment by breeswish** [comment]
CI failed. Please fix CI.

Please also add sign, sqrt into https://github.com/tikv/copr-test/blob/master/push-down-test/functions.txt by proposing a PR in the copr-test repository, and then run integration test against that PR to make sure that your implementation is the same as TiDB.

**Pull request merged**